components/event-sources/reconciler/httpsource/reconcile_test.go

```diff
@@ -435,6 +435,9 @@ func newService() *servingv1alpha1.Service {
 						Annotations: map[string]string{
 							autoscaling.MinScaleAnnotationKey: "1",
 						},
+						Labels: map[string]string{
+							"kyma-project.io/dashboard": "event-mesh",
```

Comment:
(nit) You could use your constants here. The idea is the have an object that is identical to what the "real" _make_ function generates. Not check the value of constants. It's a minor, and the code is fine regardless. Sorry for not spotting earlier 🙏